contribute/index.md

```diff
@@ -7,31 +7,26 @@ toc: true
 toc_sticky: true
 ---
 
-We warmly welcome external contributions to  ROOT! By providing code, you agree to
-transfer your copyright on the code to the "ROOT project". Of course you will be duly
-credited and your name will appear on the
-[contributors]({{'/contribute/contributors' | relative_url}}) page and in the
+We warmly welcome your contribution to ROOT!
+
+By providing code, you agree to transfer your copyright on the code to the "ROOT project".
+Of course you will be duly credited: for sizable contributions your name will appear in the
 [CREDITS](https://raw.githubusercontent.com/root-mirror/root/master/README/CREDITS){:target="_blank"}
-file shipped with every binary and source distribution. The copyright transfer is necessary
-for us to be able to effectively defend the project in case of litigation.
+file shipped with every binary and source distribution.
+The copyright transfer helps us with effectively defending the project in case of litigation.
 
-Presently, there are two ways in which you can contribute:
+There are two ways in which you can contribute:
 
 1. **Via the Users' contribution section in the Forum** <br>
    The Users' Forum has a [section](https://root-forum.cern.ch/c/my-root-app-and-ideas){:target="_blank"}
-   which describes how to submit contributions. It is the
+   which describes how to submit sizable feature contributions. It is the
```

Comment:
Does my-root-app-and-ideas really describe "how to submit sizable feature contributions"?

It's more a space for ROOT-related advertisement than for _contribution_, isn't it?

Reply:
I believe this is referring to "Guidelines for submitting a contribution" (the mechanism Vassil once introduced)

Reply:
What about now?

Reply:
:+1: